roles/keycloak/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
     module: file
     dest: "{{ keycloak_local_download_dest }}"
     state: directory
+  when: not keycloak_archive_on_target
```

Comment:
This local directory is also used as the local destination when generating a pkcs12 file when PEM files are provided.  Perhaps we can just AND together multiple conditions here so we catch that case as well.